integration-tests/src/test/java/oracle/weblogic/kubernetes/ItFmwMiiDomain.java

```diff
@@ -140,6 +150,10 @@ public static void initAll(@Namespaces(3) List<String> namespaces) {
 
   /**
    * Create a basic FMW model in image domain.
+   * Create the FMW domain with introspectVersion
+   * After domain is created and introspector exists restart the operator.
```

Comment:
```suggestion
   * After domain is created and introspector exits restart the operator.
```

Reply:
Here we would only restart the operator after introspector pod "exists" and using the long running FMW introspector to verify the usecase.